client/galaxy/scripts/mvc/library/library-dataset-view.js

```diff
@@ -652,7 +652,7 @@ var LibraryDatasetView = Backbone.View.extend({
                 <!-- BREADCRUMBS -->
                 <ol class="breadcrumb">
                     <li class="breadcrumb-item">
-                        <a title="Return to the list of libraries" href="#">Libraries</a>
+                        <a title="Return to the list of libraries" href="javascript:void(0)">Libraries</a>
```

Comment:
In some cases, like this one, you are actually removing a link to the 'base' of hashed url, breaking the current functionality (return to the list of libraries).

Reply:
Good catch. I did not realize these were actual links and not buttons. I will fix this. For a scenario like this would you prefer to keep using ```href="#"``` or instead use ```href=""```. Both are valid.

I also now just noticed that there are some places where we are currently using ```href=""``` for buttons and then returning false in the click handler to prevent the href from reloading the page. For consistency reasons, I plan on changing these to use ```javascript:void(0)```.

Reply:
I have no preference as long as the functionality is consistent. :)

Reply:
Ok, I don't have a preference either. I kept the ```href="#"``` just in case since we know it currently works. I have updated the PR to fix the issue. I have tested most of the links and they are working. I will still do more testing just to be sure. Let me know if you find any problems  with this updated PR. Thanks!